test/src/page.spec.ts

```diff
@@ -545,39 +545,79 @@ describe('Page', function () {
     it('should work', async () => {
```

Comment:
was the change here needed for the text handlers?

Reply:
do we add more objects to the main world now?

Reply:
There are some sets in the injected code, and since we always create a handle for the injected code, it's probably queryable as well. An unfortunate side-effect.

Reply:
WDYT about changing the test to use a custom class? E.g., evaluate(`class MyClass {}`) and then query objects with MyClass.prototype?

Reply:
I thought about this. It sounds reasonable! Let's do it.